src/tools/overlaybd-commit.cpp

```diff
@@ -60,7 +60,7 @@ int main(int argc, char **argv) {
     app.add_option("--algorithm", algorithm, "compress algorithm, [lz4|zstd]");
     app.add_option(
            "--bs", block_size,
-           "The size of a data block in KB. Must be a power of two between 4K~64K [4/8/16/32/64])");
+           "The size of a data block in KB. Must be a power of two between 4K~64K [4/8/16/32/64])")->default_val(0);
```

Comment:
This description conflicts: must be [4/8/16/32/64], but default is 0, the default is not in [4/8/16/32/64].

Reply:
Fixed. Initialized `block_size` to -1 in order to show a warning message while setting `--bs` without `-z`. It will change to the default value [here](https://github.com/containerd/overlaybd/pull/136/files#diff-abe1d8f469f69ca05d542c840c37206b8ecdd1323acc9995ea30dc6d7cbe899bR94-R95).